distribution/oceanbase-todo/README.md

```diff
@@ -0,0 +1,29 @@
+# OceanBase Todo List
+
+OceanBase Todo List, an extremely simple todo list web application that shows how to use ob-operator.
```

Comment:
This not not a demo for ob-operator but an application using OceanBase as backend

Reply:
OK